fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -238,6 +242,18 @@ public DataApiRequest(
 
         this.having = DruidHavingBuilder.buildHavings(this.havings);
 
+
+        if (
+                sorts != null &&
+                        !sorts.isEmpty() &&
+                        sorts.toLowerCase(Locale.ENGLISH).contains(DATE_TIME_STRING.toLowerCase(Locale.ENGLISH))
```

Comment:
if we use contains, we can educate user with proper error message. If we use starts with, error message would be 'dateTime is not metric......'. It may restrict the user to think sort is allowed only for metrics.